### DISCUSSION TITLE
Can't Edit book because edit button invisible

### DISCUSSION THREAD

**Discussion by Xiamu-ssr** [GitbookIO/community#336] [0. General bugs and questions]
![image](https://github.com/GitbookIO/community/assets/77220168/7795afc1-9f41-4808-adcf-8ec69ffb1ee1)

The original button should have appeared on the right side of the top, but now it seems to have been crowded out by many avatars due to web formatting issues

**Comment by aikoii** [comment]
Hi @Xiamu-ssr, sorry you are experiencing this issue, we are working on a fix 🙏.   Can you try restarting your whole browser?

Related to #335

**Reply by aikoii** [reply]
Hey @Xiamu-ssr, this issue should be solved 🤞. Let us know if you have trouble!
